src/DZ06/ArrayValueCalculator.java

```diff
@@ -0,0 +1,61 @@
+package DZ06;
+import  java.util.logging.Logger;
+
+import static java.lang.Integer.parseInt;
+
+public class ArrayValueCalculator{
+   int summaArray;
+   int counter;
+    private static final Logger logger = Logger.getLogger(ArrayValueCalculator.class.getName());
+    static class ArraySizeException extends Exception{
```

Comment:
You don't need a static class, it crates issues there. A typical separate class is better